connector/oidc/oidc.go

```diff
@@ -130,6 +130,8 @@ func (c *Config) Open(id string, logger log.Logger) (conn connector.Connector, e
 		scopes = append(scopes, "profile", "email")
 	}
 
+	scopes = append(scopes, "offline_access")
```

Comment:
Is this something that it would make sense to set in the client config? This is maybe outside my understanding of OIDC, but I'm just looking at l. 128, `scopes = append(scopes, c.Scopes...)` and wondering if it would make sense to use that?